README.md

```diff
@@ -8,7 +8,17 @@ A [collection of companies using Elixir](https://elixir-companies.com/) in produ
 
 Proudly built with [Phoenix](https://phoenixframework.org).
 
-## Running
+### Adding a new company to the list
```

Comment:
We can also add jobs on companies, can you mention that? Thank you :)

Reply:
@gemantzu Done!